modules/@apostrophecms/doc-type/ui/apos/components/AposDocEditor.vue

```diff
@@ -161,7 +163,7 @@ export default {
       live: null,
       published: null,
       errorCount: 0,
-      restoreOnly: false
+      archived: false
```

Comment:
I renamed this because "restore" is no longer the only thing you can do to an archived item in the editor; you can also delete it permanently.